pages/portfolio.js

```diff
@@ -46,39 +46,42 @@ class Portfolio extends React.Component {
       .map((work) => work.category.main)
       .reduce((a, b) => a.concat(Array.isArray(b) ? flatten(b) : b), []); // flatten
     const uniqCategories = [...new Set(categories)];
-
-    const selectedWorks = chosenCategory ? transformateCategories(chosenCategory.split(','), uniqCategories)
-      : uniqCategories;
-
-    return { selectedWorks, categorisList: uniqCategories };
+    // no selected categories by default
+    const selectedCategories = chosenCategory ? transformateCategories(chosenCategory.split(','), uniqCategories)
+      : [];
+    return { selectedCategories, categoriesList: uniqCategories };
   }
 
   worksCountFor(work) {
-    const { selectedWorks } = this.state;
-    return work.category.main.filter((category) => selectedWorks.includes(category)).length;
+    const { selectedCategories } = this.state;
+    // When no categories selected - show all works
+    if (!selectedCategories.length) {
```

Comment:
```suggestion
    if (selectedCategories.length === 0) {
```

Reply:
for the consistency (see previous file, line 97)

Reply:
Done